File: Proyecto Seguridad Informatica/backend/src/main/java/com/seguridadi/backend/model/Activo.java
package com.seguridadi.backend.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

import lombok.Data;

@Entity
@Data
public class Activo {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String tipo;
    private String descripcion;
    private Double valor;
    private String categoria;
    private String vulnerabilidad;
    private String impacto;

}

File: Proyecto/valoracion-activos/server/models/Activo.js
const mongoose = require('mongoose');

const ActivoSchema = new mongoose.Schema({
    tipo: { type: String, required: true },
    descripcion: { type: String, required: true },
    categoria: { type: String, required: true },
    valoracion: { type: Number, required: true },
    riesgo: { type: Number, required: true },
    impacto: { type: Number, required: true }
});

module.exports = mongoose.model('Activo', ActivoSchema);
